docs/sources/alerting/unified-alerting/opt-in.md

```diff
@@ -7,8 +7,12 @@ description = "How to enable Grafana 8 Alerts"
 Setting the `ngalert` feature toggle enables the new Grafana 8 Alerting system.
 
 >**Note:** It is recommended to backup Grafana's database before enabling this feature.
+>**Note:** If you use PostgreSQL as a backend database, for using Grafana 8 Alerts, the minimum required version is 9.5.
```

Comment:
```suggestion
>**Note:** It is recommended to backup Grafana's database before enabling this feature. If you are using PostgreSQL as the backend data source, the minimum required version is 9.5.
```

Reply:
I suggest also adding a link to instructions for backing up the database.